tests/excerptexport/views/test_export_list.py

```diff
@@ -41,21 +41,19 @@ def setUp(self):
             orderer=User.objects.create_user('another_user', 'someone_else@example.com', 'top secret')
         )
 
-    def test_permission_denied_if_lacking_email(self, *args):
+    def test_access_ok_even_without_confirmed_email(self, *args):
```

Comment:
Should we add a test for testing without logged in user, right now we only test happy path - one could argue, that the logged in or not is being tested inside the Django framework itself, so no need to test it here.